### ISSUE TITLE
Include examples in hardware requirements

### ISSUE THREAD

**Issue by apaskulin** [sensu/sensu-docs#1099]
Page: https://docs.sensu.io/sensu-go/5.1/installation/recommended-hardware/

To do:

- Under backend recommended configuration, add an example such as: For x # of servers with x # of checks for each server every minute (total of x # of checks per minute, x # of checks per second), we recommend x # of cores.
- Update the number of recommended cores for the agent from four to one.
- Include equivalent information to: If you're deploying Sensu with Docker and a cloud provider, we recommend using a high-performance storage service like Amazon Elastic Block Store (with provisioned IOPS) or Google Persistent Disk. If you're using spinning disks, ensure you are using throughput optimized HDDs.

Reference: https://coreos.com/etcd/docs/latest/op-guide/hardware.html

**Comment by hillaryfraley** [comment]
The recommended hardware doc already addresses item 1 under https://docs.sensu.io/sensu-go/latest/installation/recommended-hardware/#backend-recommended-configuration.

For item 2, Eng suggests keeping the recommendation at four cores.

So I think the first two items are handled, and I'm wondering if everyone else agrees that we only need to address the third item to resolve this issue.

**Comment by hillaryfraley** [comment]
For third item, add under https://docs.sensu.io/sensu-go/latest/installation/recommended-hardware/#cloud-recommendations.

Need to identify the size to recommend. This is on hold for now because we are working with Engineering to identify arch recommendations.

**Comment by hillaryfraley** [comment]
Link to performance testing via Loadit tool (see https://github.com/sensu/sensu-go/issues/4175). Issue currently scheduled for ~6.5.0~ ~6.6.0~ 6.next.

Once the readme is in place, we may need to add content about how to do a load test.

**Comment by hillaryfraley** [comment]
See also: https://github.com/sensu/sensu-docs/issues/3574

**Comment by hillaryfraley** [comment]
Moved loadit documentation to its own issue: https://github.com/sensu/sensu-docs/issues/3615
